package.json

```diff
@@ -8,7 +8,7 @@
   },
   "main": "index.js",
   "dependencies": {
-    "@bcgov/nodejs-common-utils": "0.0.2",
+    "@bcgov/nodejs-common-utils": "0.0.3",
```

Comment:
Its good practice to run `npm outdated` to learn what packages are out of date; especially when under development. In production, you tend to what to lock these down.

Reply:
Got it! will start another branch to for update.